internal/olm/operator/internal/registry_pod.go

```diff
@@ -319,3 +313,11 @@ func (rp *RegistryPod) GetLogs(ctx context.Context) (string, error) {
 	}
 	return buf.String(), nil
 }
+
+// GetRegistryPodName returns the registry pod name
+func (rp *RegistryPod) GetRegistryPodName() (string, error) {
+	if rp.pod == nil {
+		return "", errors.New("a registry pod must be created before getting pod name")
+	}
+	return rp.pod.Name, nil
+}
```

Comment:
Why  it is required? Why not use `rp.Pod.Name` instead of ave this func?

Reply:
That requires making pod as an exported field. I don't think it's a good idea to expose the entire corev1.pod. 
Instead of that, using a getter function to obtain the pod name.